src/ffi/fuzz.rs

```diff
@@ -0,0 +1,23 @@
+use crate::ffi::wasm;
+
+static STATIC_TAG: &[u8] = b"FUZZING STATIC TAG";
+
+#[derive(arbitrary::Arbitrary, Debug)]
+pub struct UnblindRespInput {
+    blinded_resp_buf: Vec<u8>,
+}
+
+pub fn unblind_resp(input: UnblindRespInput) {
+    // TODO(victor): Figure out a good way to make these values constant between calls.
```

Comment:
`static_keypair` and `static_blinded_message`. In this fuzzing routine they do not vary, so there is no reason to waste CPU cycles recalculating them each time. It is quite possible that they are calculated at compile time and optimized out, but this implementation does guarantee it and the functions are complicated enough that it may not happen.